packages/admin/cms-admin/src/dam/DamTable.tsx

```diff
@@ -175,9 +176,18 @@ export const DamTable = ({ damLocationStorageKey, ...props }: DamTableProps): Re
         }
     }, [filterApi, filterApi.current.sort, setSorting]);
 
+    let stateKey: string;
+    const scope = useDamScope();
+
+    if (Object.keys(scope).length > 0) {
+        stateKey = `${Object.values(scope).join("-")}-${damLocationStorageKey}`;
```

Comment:
Theoretically, the order of the keys in the scope object could change, but I've decided to ignore that, as we don't care about this in the ContentScopeProvider either 😁